helpers/helpers.cjs

```diff
@@ -540,3 +540,18 @@ function expandTildePath(file, os) {
   return homedir ? file.replace(/^~($|\/|\\)/, `${homedir}$1`) : file;
 }
 module.exports.expandTildePath = expandTildePath;
+
+/**
+ * Downcases a string and strips extra white spaces and punctuations
```

Comment:
Period at the end of all these, please, even though I know they're not really sentences. Holdover from .NET days...